app/course-scraper/src/config.py

```diff
@@ -1,6 +1,8 @@
 class CONFIG:
     # XXX: Modify as needed
-    DATA_PATH = r"C:\Users\user\Documents\coding\learning-path-index\live-demo"
+    DATA_PATH = r"C:\PillView\learning-path-index"
```

Comment:
@asvcode you have a hard-coded path that might not work for other users

@TobeTek this may need to change at the original implementation level maybe we can introduce an environment variable that the end-user can set

This gives them a freedom to select any location

Reply:
Can one of you pls create an issue about it and link to this PR and then we can go ahead and merge+close this PR